src/lib/network/network_autoyast.rb

```diff
@@ -113,6 +117,28 @@ def configure_lan(write: false)
       configure_submodule(Lan, ay_configuration, write: write)
     end
 
+    # Takes care of activate s390 devices from the profile declaration
+    def activate_s390_devices(section = nil)
+      profile_devices = section || ay_networking_section["s390-devices"] || {}
+      devices_section = Y2Network::AutoinstProfile::S390DevicesSection.new_from_hashes(profile_devices)
+      connections = Y2Network::Autoinst::S390DevicesReader.new(devices_section).config
+      connections.each do |conn|
+        begin
+          builder = Y2Network::InterfaceConfigBuilder.for(conn.type, config: conn)
+          activator = Y2Network::S390DeviceActivator.for(builder)
+          if activator.configure
+            builder.name = activator.configured_interface
```

Comment:
what is the goal is assigning name to builder when you do not save it?

Reply:
It is not needed at all. It makes sense when activating the device for editing, but in AY it is irrelevant.